service/src/main/kotlin/io/provenance/explorer/service/async/ScheduledTaskService.kt

```diff
@@ -388,4 +393,9 @@ class ScheduledTaskService(
             }
         }
     }
+
+    @Scheduled(initialDelay = 1L, fixedDelay = 5L, timeUnit = TimeUnit.MINUTES)
```

Comment:
at some point, I'd like to see these scheduled tasks broken up into separate files rather than one huge ScheduledTaskService. There is just too much going on in that service to know the scope of a change. This addition is fine for now though, just future thought.